strings/strings.c

```diff
@@ -9,7 +9,8 @@
 */
 int string_length(char s[])
 {
```

Comment:
Look into '\0' string terminators in C.  You can loop till your get to the end of the string terminator while incrementing your counter, is an approach you can use.